test/compilable/diag13110a.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+name=`basename $0 .sh`
+dir=${RESULTS_DIR}/compilable
+output_file=${dir}/${name}.d
+
+function echorun() {
+    echo $@ > ${output_file}
+    $@
+}
```

Comment:
What's the motivation for this `echorun`?

Reply:
It runs the command and logs the command to the output_file

Reply:
Logging the output is simple with `tee`:

`| tee ${output_file} | ...`

If you want to log all arguments, `-x` is your friend.
You can even dump the entire debug stream to the output_file:

```shell
exec 42> ${output_file}
export BASH_XTRACEFD=42
```

(`42` is arbitrary)

Reply:
it logs the command, not the output of the command...how would you use `tee` for that?

Reply:
NOTE: I removed this in favor of https://github.com/dlang/dmd/pull/7928 and possible future improvements to support "logging by default" to BASH tests.